### DISCUSSION TITLE
Are they not able to track this if the attack is always frameperfect?

### DISCUSSION THREAD

**Discussion by NibbaNewt** [256shadesofgrey/eso-light-attack-weave#21] [General]
But tbh if you havent been banned after using it for years its fine right?

**Comment by 256shadesofgrey** [comment]
I have no idea what they are tracking, and I cannot promise that you won't be banned if you use it. I certainly didn't build in any countermeasures to prevent it.

But yeah, I have been using some version of this macro for years (currently running linux, so I'm using the python implementation that is linked in the readme). But then again I only play PvE, and it's much more likely that you'd get reported for it in PvP, and then they might look closer.
